pkg/ruler/storage.go

```diff
@@ -17,7 +17,7 @@ import (
 	"github.com/cortexproject/cortex/pkg/ruler/rules/objectclient"
 )
 
-// RuleStoreConfig conigures a rule store
```

Comment:
This is unrelated, but found it whilst browsing and thought it would be small enough to just pass-by within this PR.